src/borg/helpers/fs.py

```diff
@@ -65,6 +65,15 @@ def get_cache_dir():
                 """).encode('ascii'))
     return cache_dir
 
+def get_config_dir():
+    """Determine where to store whole config"""
+    xdg_config = os.environ.get('XDG_CONFIG_HOME', os.path.join(get_home_dir(), '.config'))
+    config_dir = os.environ.get('BORG_CONFIG_DIR', os.path.join(get_home_dir(xdg_config), 'borg'))
```

Comment:
there is code in get_keys_dir and get_security_dir that could get removed by calling get_config_dir (currently it has no callers, so is not especially useful).

refactor them and run the tests afterwards.